test/e2e/accounts/snap-account-eth-swap.spec.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports */
+import { withFixtures, defaultGanacheOptions, WINDOW_TITLES } from '../helpers';
+import { Driver } from '../webdriver/driver';
+import { installSnapSimpleKeyring } from './common';
+
+const FixtureBuilder = require('../fixture-builder');
+const {
+  buildQuote,
+  reviewQuote,
+  waitForTransactionToComplete,
+  checkActivityTransaction,
+} = require('../tests/swaps/shared');
+
+describe('Snap account swap', function () {
+  it('exchanges native token (ETH) for DAI', async function () {
```

Comment:
Give the previous comment, we can just call this
```suggestion
  it('swaps ETH for DAI using a snap account', async function () {
```

Reply:
Updated, thanks!